plugins/battery/battery.plugin.sh

```diff
@@ -6,7 +6,7 @@ if ! command -v "command_exists" >/dev/null; then	command_exists() { command -v
 ac_adapter_connected(){
   if command_exists upower;
   then
-    upower -i $(upower -e | grep BAT) | grep 'state' | grep -q '[^(dis)]charging\|fully-charged'
+    upower -i $(upower -e | grep BAT) | grep 'state' | grep -q '[: \t]charging\|fully-charged'
```

Comment:
Thank you for the update! However, there is still an issue. In the POSIX regular expressions, you cannot use `\t` in bracket expressions to represent a TAB character except for `awk`, i.e., `[: \ t]` matches one of `:`, <code>&nbsp;</code> (space), `\`, or `t`.